client-app/shared/static-content/components/slider.vue

```diff
@@ -28,16 +30,63 @@
 </template>
 
 <script setup lang="ts">
+import { useBreakpoints } from "@vueuse/core/index";
 import { Navigation } from "swiper";
-import { Swiper, SwiperSlide } from "swiper/vue"; // eslint-disable-line import/no-unresolved
+import { Swiper, SwiperSlide } from "swiper/vue";
 import { computed, getCurrentInstance } from "vue";
+import { BREAKPOINTS } from "@/core/constants";
```

Comment:
import { breakpointsTailwind, useBreakpoints } from '@vueuse/core'

const breakpoints = useBreakpoints(breakpointsTailwind)

Reply:
@goldenmaya See #677
Breakpoints was already declared (and we used all of them) and they have one additional breakpoint (xs) and slightly different values (2xl = 1500px instead of 1536px)